src/components/Counter.js

```diff
@@ -1,16 +1,22 @@
-import React, { Component } from "react";
+import React from "react";
 import { connect } from 'react-redux';
 import { increment, decrement } from '../actions';
+import { EventEmitter } from "events";
```

Comment:
I'm getting a warning about this package never being used. What was your plan for this? :thinking: